.editorconfig

```diff
@@ -23,6 +43,10 @@ indent_size = 2
 indent_style = space
 indent_size = 2
 
+# FIXME this needs to be gone eventually
```

Comment:
If I understand it correctly, the Makefile syntax requires tabs and the precompiler breaks on lines when they do contain tabs, but spaces are fine?